mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/BrukerRoutes.kt.kt

```diff
@@ -38,18 +38,17 @@ fun Route.brukerRoutes() {
 
     route("/api/v1/bruker/historikk") {
         get {
-            poaoTilgangService.verifyAccessToUserFromVeileder(getNavAnsattAzureId(), getNorskIdent())
-            historikkService.hentHistorikkForBruker(getNorskIdent())?.let {
+            poaoTilgangService.verifyAccessToUserFromVeileder(getNavAnsattAzureId(), getNorskIdent()) {
+                auditLog.log(createAuditMessage("NAV-ansatt med ident: '${getNavIdent()}' forsøkte, men fikk ikke sett tiltakshistorikken for bruker med ident: '${getNorskIdent()}'."))
+                call.respondText(
```

Comment:
Er denne nødvendig når `verifyAccessToUserFromVeileder` uansett kaster exception?

Reply:
Niet, vi fjerner den og erstatter med en securelog for feilsøking dersom en veileder ikke har tilgang.